src/lib.rs

```diff
@@ -19,6 +19,8 @@ mod lib {
 }
 
 mod conversion;
+#[cfg(feature = "num-traits")]
+pub mod num_traits_impls;
```

Comment:
Does the module need to be public? It just contains trait impls so nothing to export, right?